src/js/gi/actions/index.js

```diff
@@ -63,21 +64,27 @@ function withPreview(dispatch, action) {
       type: ENTER_PREVIEW_MODE,
       version
     });
+  } else if (version.createdAt) {
+    dispatch({
+      type: SET_VERSION,
+      version
+    });
   }
+
   dispatch(action);
 }
 
-export function fetchConstants() {
-  const previewVersion = ''; // TODO
-  const url = `${api.url}/calculator_constants?preview=${previewVersion}`;
+export function fetchConstants(version) {
+  const queryString = version ? `?version=${version}` : '';
+  const url = `${api.url}/calculator_constants${queryString}`;
 
   return dispatch => {
     dispatch({ type: FETCH_CONSTANTS_STARTED });
 
     return fetch(url, api.settings)
       .then(res => res.json())
       .then(
-        payload => dispatch({ type: FETCH_CONSTANTS_SUCCEEDED, payload }),
+        payload => withPreview(dispatch, { type: FETCH_CONSTANTS_SUCCEEDED, payload }),
```

Comment:
Currently, requests for constants will always return `version.preview` as `false`, since there's no versioning on constants at the moment. This is mainly to set the (otherwise null) date on the preview banner, since the landing page doesn't make any other API request, and we need to show the banner on every GIBCT page as long as there is a `version` query param.